ci/toolchain_manager.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
+
+# Builds and publishes kubesaw components to a quay repository.  Used as a part
+# of the CI process for testing in openshift-ci.
+
+set -e
+
+# parse input
+BRANCH=${BRANCH:-pubviewer-mvp}
```

Comment:
Why this branch and not main, is this temporary?

Reply:
This branch contains what we use for testing, since it has public viewer support across the various components.  Once that gets merged, we can go back to using `main`.

Reply:
OK